fe1-web/src/core/objects/Timestamp.ts

```diff
@@ -88,4 +90,37 @@ export class Timestamp extends Number implements Number {
   public addSeconds(seconds: number): Timestamp {
     return new Timestamp(this.valueOf() + seconds);
   }
+
+  /**
+   * Returns *some* string representation of this object.
+   * If you need access to the unterlying data type use .valueOf() and
+   * if you want to serialize an instance use .toState() instead
+   */
+  public toString(): string {
+    return super.toString();
+  }
+
+  /**
+   * Returns the primitive value used for representing the Timestamp,
+   * the number of seconds that passed since 00:00:00 UTC
+   * If you want to serialize an instance use .toState() instead
+   */
+  public valueOf(): number {
+    return super.valueOf();
+  }
+
+  /**
+   * Returns the serialized version of the base64url that can for instance be stored
+   * in redux stores
+   */
+  public toState(): TimestampState {
+    return super.valueOf();
+  }
```

Comment:
In this particular case, though, I fully agree with implementing the `TimestampState`.
The number is indeed an arbitrary representation of a timestamp, and we could instead use ISO-8601.

Reply:
Yes, I will do that actually :)